Core/Core/View/Base/DownloadView.swift

```diff
@@ -14,16 +14,29 @@ public enum DownloadViewState {
     case finished
 }
 
+
+public struct CircleProgressView: View {
+    public init () {
+    }
+
+    public var body: some View {
+        ProgressBar(size: 30, lineWidth: 1.75)
+    }
+}
```

Comment:
The `ProgressBar` itself is circular, then why its needed to define it again? Can't we use it directly, like we are using it in the whole codebase?